resources/ceph_block_pool.py

```diff
@@ -0,0 +1,51 @@
+"""
+General Ceph block pool class
+"""
+import yaml
+import logging
+from resources.base_resource import BaseOCSClass
+from ocs import defaults
+from utility.templating import load_yaml_to_dict
+
+
+log = logging.getLogger(__name__)
+
+
+class CephBlockPool(BaseOCSClass):
+    """
+    CephBlockPool kind class
+    Provides basic methods to operate on CephBlockPool kind.
+    """
+    def __init__(self, **kwargs):
+        """
+        Initializer method
+
+        kwargs:
+            Copy of ocs/defaults.py::CEPHBLOCKPOOL_DICT dictionary
+        """
+        self.cbp_data = defaults.CEPHBLOCKPOOL_DICT
```

Comment:
copy()

Reply:
@RazTamir Actually, it would be better if we do the copy outside, like done here - https://github.com/red-hat-storage/ocs-ci/pull/132/files

I will do this here too

Reply:
I'm not sure which part I should look for but in the next line, you are changing the original dict